package.json

```diff
@@ -162,7 +162,8 @@
     "setupFilesAfterEnv": [
       "<rootDir>/dev/jest.setupTestFrameworkScriptFile.js"
     ],
-    "testEnvironment": "jest-environment-jsdom-global"
+    "testEnvironment": "jest-environment-jsdom-global",
+    "testURL": "http://test-url.com"
```

Comment:
Could you clarify why this change is needed? I don't understand it's impact in the contact of this PR.

Reply:
I added a test url to give jsdom a consistent url in the tests, before I added this I undefined as the url in the tests

Reply:
On the prebid tests?

Reply:
Spoke to Damian about this 'offline' (which is actually online because everyone's WFH) and we're reached a healthy compromise. Yay! 🎉